src/Metadata/Driver/XmlDriver.php

```diff
@@ -119,7 +120,12 @@ protected function loadMetadataFromFile(\ReflectionClass $class, string $path):
                 $discriminatorGroups[] = (string) $entry;
             }
 
-            $metadata->setDiscriminator($discriminatorFieldName, $discriminatorMap, $discriminatorGroups);
+            $metadata->setDiscriminator(
+                $discriminatorFieldName,
+                $discriminatorMap,
+                $discriminatorGroups,
+                !empty($discriminatorDefaultClass) ? $discriminatorDefaultClass : null,
```

Comment:
Let's make it a bit shorter. 
```suggestion
               $discriminatorDefaultClass ?? null,
```